Formula/helidon.rb

```diff
@@ -0,0 +1,21 @@
+class Helidon < Formula
+  desc "Set of Java libraries for writing microservices that run on Netty"
+  homepage "https://helidon.io/"
+  url "file:///Users/tvallin/workspace/helidon-build-tools/cli/impl/target/distribution/helidon.zip"
```

Comment:
We need to stage the distribution in helidon.io ; that means a change in `helidon-site-files`.
We can reference the Maven Central URL now: https://repo1.maven.org/maven2/io/helidon/build-tools/cli/helidon-cli-impl/3.0.4/helidon-cli-impl-3.0.4.zip

Reply:
Had to change the url because of homebrew complaining, but it works fine.